spec/std/isa/csr/Ssaia/stopi.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Kallal Mukherjee.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: stopi
+long_name: Supervisor Top Interrupt
+address: 0x15B
+writable: true
```

Comment:
Per 5.4.2 Supervisor top interrupt CSR (`stopi`) (and on line 16 in this file, below):
> Supervisor-level CSR stopi is read-only